SportsHub/SportsHub.Core/ControllerConstants/LoginControllerConstants.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace SportsHub.Domain.ControllerConstants
+{
+    public class LoginControllerConstants
```

Comment:
Move every Constant class into one file, also I don`t think that we need separate classes for every constant, something like ControllerConstants is enough. The constant class should be Static. and if there are constants that related only to the API level, then move it there